roles/os_hardening/templates/etc/pam.d/rhel_auth.j2

```diff
@@ -47,6 +47,9 @@ password    required      pam_deny.so
 session     optional      pam_keyinit.so revoke
 session     required      pam_limits.so
 -session    optional      pam_systemd.so
+{% if (os_auth_pam_oddjob_mkhomedir | bool) %}
+session     optional      pam_oddjob_mkhomedir.so umask=0077
```

Comment:
sorry for being late to the party.
We have a special variable fo setting the expected umask (`os_env_umask`). While checking for this, I also found, that this setting might be ignored, because of https://github.com/authselect/authselect/issues/223

So no real problem here, but a minor inconvenience from my point of view.